src/Sylius/Component/Product/Model/ProductVariant.php

```diff
@@ -108,9 +108,9 @@ public function removeOptionValue(ProductOptionValueInterface $optionValue): voi
         }
     }
 
-    public function hasOptionValue(ProductOptionValueInterface $optionValue): bool
+    public function hasOptionValue(?ProductOptionValueInterface $optionValue): bool
     {
-        return $this->optionValues->contains($optionValue);
+        return $optionValue === null ? true : $this->optionValues->contains($optionValue);
```

Comment:
Why true instead of false?

Reply:
this is abit tricky, because by returning true we force it ProductVariant to dont allow null as argument and handle it as bad request.

Reply:
It is tricky because it feels wrong to check whether a product variant has an option value `null`. I think we should fix it upstream and do not let `null` to be passed to this method in the first place.